docker-compose.test.yml

```diff
@@ -0,0 +1,33 @@
+version: "2.1"
+services:
+  postgres:
+    image: postgres:14-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: docker
+      POSTGRES_PASSWORD: docker
+      POSTGRES_DB: postgres
+    ports:
+      - "5432"
+      - "5431:5432"
+    networks:
+      - internal
+  sut:
+    image: balena/pinejs-sut
+    command: npm run mocha
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: sut
+    depends_on:
+      - postgres
+    environment:
+      LOGLEVEL: error
+      API_HOST: 127.0.0.1 
+      DATABASE_URL: postgres://docker:docker@postgres:5432/postgres 
+      PORT: 80 
```

Comment:
Is this used?

Reply:
DATABASE_URL is used, the rest will be removed.